src/javaRosa.js

```diff
@@ -2102,12 +2102,12 @@ define([
                 {
                     name: "Edit Bulk Translations",
                     action: function (done) {
-                        _this.showItextDialog(done);
+                        _this.showItextModal(done);
                     }
                 }
             ]);
         },
-        showItextDialog: function (done) {
+        showItextModal: function (done) {
```

Comment:
No good reason. For the most part, we've ended up using "dialog" for functions that use the jQuery UI widget and "modal" for the bootstrap widget, so my todo list for this PR was basically "grep -i dialog", and I ended up changing this.